src/plugins/core/sheet.ts

```diff
@@ -176,7 +176,7 @@ export class SheetPlugin extends CorePlugin<SheetState> implements SheetState {
       case "CREATE_SHEET":
         const sheet = this.createSheet(
           cmd.sheetId,
-          cmd.name || this.getNextSheetName(),
```

Comment:
I'd keep the `|| this.getNextSheetName()` because we still have revisions without the name in the wild